src/components/CreateMappingModal.vue

```diff
@@ -94,8 +94,8 @@ export default defineComponent({
     })
   },
   methods: {
-    closeModal() {
-      modalController.dismiss({ dismissed: true });
+    closeModal(payload = {}) {
```

Comment:
We can pass the value as a direct parameter instead of passing it as an object, as we have single data to be passed here, hence passing an object and then spreading its value is not a good approach here.